wallet/wallet.go

```diff
@@ -1481,9 +1481,9 @@ type Balances struct {
 // outputs must be iterated.
 func (w *Wallet) CalculateAccountBalances(account uint32, confirms int32) (Balances, error) {
 	var bals Balances
-	err := walletdb.View(w.db, func(tx walletdb.ReadTx) error {
+	err := walletdb.Update(w.db, func(tx walletdb.ReadWriteTx) error {
```

Comment:
Kinda weird now that something like just reading balances now can mutate the database and requires a write transaction.

Reply:
No longer an issue.